File: TPraticas/Aula11.a/codigofonte/ReadTemps.py
temps = [None]*10
numTemps = 0
infile = open("/tmp/temps.txt","r")           
for line in infile.readlines():         
     for i in line.split():            
         temps[numTemps] = float(i)   
         numTemps += 1
print 'Foram lidas ' + str(numTemps) + ' temperaturas.'

File: TPraticas/Aula11.a/codigofonte/ReadTemps.java

import java.io.*;  // File
import java.util.*;  // Scanner

public class ReadTemps
{
    public static void main (String[] args) throws FileNotFoundException
    {
      Scanner inFile = new Scanner(new File("/tmp/temps.txt"));  
      double[] temps = new double[10];
      int numTemps = 0;

      while (inFile.hasNextDouble())
      {
        temps[numTemps] = inFile.nextDouble();
        numTemps++;
      }
      System.out.println("Foram lidas " + numTemps + " temperaturas.");

    }
}